chart/kubeapps/values.yaml

```diff
@@ -437,6 +437,9 @@ dashboard:
   ## ref: https://github.com/kubeapps/kubeapps/blob/master/docs/developer/custom-form-component-support.md
   ##
   customComponents: ""
+  ## @param dashboard.remoteComponentsUrl Remote URL that can be used to load custom components vs loading from the local filesystem
```

Comment:
This param should also appear in the readme, not a big deal as we always re-generate the chart readme before releasing.

If curious, we use a bitnami tool for that. Here are the steps I use to follow:

1. `git clone https://github.com/bitnami-labs/readme-generator-for-helm.git`
2. ` cd readme-generator-for-helm/`
3. ` npm install`
4. `node bin/index.js  -r ../kubeapps/chart/kubeapps/README.md -v ../kubeapps/chart/kubeapps/values.yaml `

Reply:
Im sorry u mentioned this last time as well, will generate when I adjust test